library/quality/quality.gradle

```diff
@@ -69,8 +69,7 @@ task pmd(type: Pmd) {
 
 android {
     lintOptions {
-        disable 'InvalidPackage',
-                'NewerVersionAvailable' // Comment this line out to do a quick check for library version updates.
+        disable 'InvalidPackage', 'RestrictedApi' // TODO remove RestrictedApi
```

Comment:
There seems to be a bug in debug builds where we get a bunch of `RestrictedApi` errors.